### PR TITLE
Support for types of multi match query

### PR DESCRIPTION
**Pull request by npatmaja**
This PR adds support for specifying [types of multi match query](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-multi-match-query.html#multi-match-types).

This is done by adding another argument into `multiMatchQuery` called `type` that defaulted to `best_fields`, which is the default value in elastic search. The following code is an example of the use case:

```js
bodybuilder.query('multiMatch', ['field1', 'field2'], 'keyword', 'phrase_prefix');
```

### CODE REVIEW

File: src/queries/multi-match-query.js [hunk 1]
```diff
@@ -3,12 +3,14 @@
  *
  * @param  {Array}  fields  The field names to query over.
  * @param  {String} query   The query string.
+ * @param  {String} type    The type of multi_match query (optional).
```
Comment (danpaz) [inline src/queries/multi-match-query.js]: Nit: follow the jsdoc convention for optional params http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values.

```
@param {String} [type='best_fields'] The type of multi_match query.
```

**Comment by danpaz** [comment]
LGTM. Just one nit.

**Comment by npatmaja** [comment]
@danpaz done!

**Comment by danpaz** [comment]
Thanks @npatmaja ! 😃

**Pull request merged**